### PR TITLE
pkg/control: Add a few metrics to monitor steady state better

### PR DESCRIPTION
**Pull request by evanphx**

### CODE REVIEW

File: cmd/hzn/main.go [hunk 1]
```diff
@@ -143,6 +144,12 @@ func StartHealthz(L hclog.Logger) {
 		w.WriteHeader(200)
 	})
 
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```
Comment (mitchellh) [inline cmd/hzn/main.go:151]: In our other products we usually hide these behind AuthN/AuthZ. Should we perhaps do the same here?

Comment (evanphx) [reply cmd/hzn/main.go:151]: These are only running on the metrics endpoint, which is never exposed.

Comment (evanphx) [reply cmd/hzn/main.go:151]: Metrics run a separate port with separate http listener.

Comment (mitchellh) [reply cmd/hzn/main.go:151]: Got it!

**Review by mitchellh — APPROVED** [review]

**Pull request merged**